src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +32,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /** The feedback message to be shown to the user. Contains a description of the execution result */
```

Comment:
Since it is very trivial that you get feedbackToUser object using this method, you can skip writing a method header comment. In case If you would like to keep it, please always begin the comment with the plural word to be adhere to the coding standard. E.g. _Returns FeedbackUserObject_